src/guide/reviewing-and-sponsoring-a-change.md

```diff
@@ -0,0 +1,73 @@
+# Reviewing and Sponsoring a Change
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [GitHub Docs on pull requests reviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews)
+:::
+
+Two of the most important contributions you can make in the OpenJDK community are to review and sponsor changes for other developers. All changes needs to be reviewed. Many changes even need to be reviewed by more than one person. This means that in order to enable a fast-pased development experience, all developers need to review more changes then they produce themself.
+
+If you are new to an area, reviewing changes is also a great way to learn the code and see what general styles and types of changes that are relevant in the area. Be mindful though, if you don't know the area well you should state this in your review comments. As a **R**eviewer you have a responsibility to make sure changes are sound and goes in line with the general direction of the area. If you, as a **R**eviever, review a change in an area that you don't know well you probably shouldn't be the one to approve the change.
```

Comment:
s/changes that/that/
s/goes in line/go in line/ or perhaps better just "and align with ..."